PD-1674 / 25.10 / PD-1674-custom-app-ui-ref-update (#3549)
truenas/documentation: static/includes/apps/CustomAppIntro.md

To deploy a custom application, go to **Apps** and click **Discover Apps**.
Click **Custom App** to open the **[Install iX App]({{< relref "installcustomappscreens.md #install-ix-app-screen" >}})** screen with a guided installation wizard.
Click <i class="material-icons" aria-hidden="true" title="more_vert">more_vert</i> > **Install via YAML** to open the **[Custom App]({{< relref "installcustomappscreens.md #custom-app-screen" >}})** screen with an advanced YAML editor for deploying apps using Docker Compose.

TrueNAS also allows users to convert already installed apps into custom applications. 

> Click on the app to wish to convert, and then click the widget next to **Edit** to open the menu containing **Update** and **Convert to custom app** options.

Reviewer: Is this the Edit button on the Applications Info widget? if it is, say that. Users might be confused but this sentence. How about this, if it is a correct interpretation of what you are saying here:
```suggestion
Select the row for the app to wish to convert, and then click **Edit** on the **Applications Info** widget.
The dropdown list with **Update** and **Convert to custom app** shows.
```

{{< trueimage src="/images/SCALE/Apps/ConvertToCustomAppMenu.png" alt="Convert to Custom App Menu" id="Convert to Custom App Menu" >}}

> After selecting **Convert to custom app** from the menu, users can confirm the conversion on a confirmation screen.

Reviewer: Style guide says to use dropdown list and not menu. It is okay to talk about the confirmation dialog or leave it out and shorten it by just saying select Confirm. How about this for clarity:
```suggestion
Select **Convert to custom app** from the dropdown list, then select **Confirm** to convert the app. 
```

{{< trueimage src="/images/SCALE/Apps/ConvertToCustomAppConfirm.png" alt="Convert to Custom App Confirm" id="Convert to Custom App Confirm" >}}

> Once converted, editing the information of a custom app opens an **Edit App YAML** window that gives users the option to enter a custom app configuration.

Reviewer: Per the style guide, don't use once, use after. Once mead do it one time, where after says after something happens now do this. Also this sentence is not making sense. What is happening here? after you select confirm where do you edit  the information? Is it in the Edit App YAML window? This sentence is not clear on what the user does next.
